### PR TITLE
Release 0.1

### PR DESCRIPTION
**Pull request by Qkley**
This is the release-0.1 pull request. I have made a ton of changes and tried to make the functions look consistent. Could you please check thoroughly and make sure that compulsory elements for this release are included? 

I look forward to your comments and suggestions.

### CODE REVIEW

File: viziphant/rasterplot.py [hunk 1]
```diff
@@ -605,6 +614,31 @@ def sort_func(x):
...
+
+    Returns
+    -------
+    ax : matplotlib.Axes.axes
+        The handle for the raster plot.
+
+    """
+    ax, axhistx, axhisty = plot_raster_rates(spiketrain_list)
+    axhistx.set_visible(False)
+    axhisty.set_visible(False)
```
Comment (dizcza) [inline viziphant/rasterplot.py]: this is called sweeping under the carpet
this function should have simple and clear _implementation_
I agree to leave it for now like it is currently, we'll rethink it in viziphant-new

File: requirements/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+neo>=0.8.0,<0.9.0
+elephant>=0.6.4
```
Comment (dizcza) [inline requirements/requirements.txt]: please check if it works with elephant-0.6.4 and elephant-0.7.0

File: setup.py [hunk 3]
```diff
@@ -1,6 +1,44 @@
...
+    long_description=long_description,
+    license="BSD",
+    url='https://github.com/INM-6/viziphant',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
```
Comment (dizcza) [inline setup.py]: we should remove this line and state somewhere that viziphant works with python 3.6+ only

**Comment by mdenker** [comment]
The install script is missing a file called `viziphant/VERSION`, which is a text file with the version number 0.1.

File: requirements/requirements.txt (hunk 2 above)
Comment (Qkley) [reply requirements/requirements.txt]: My dev environment for this release was elephant==0.6.4. I have tested now by changing it to elephant==0.7.0 and plot_isi, plot_corrcoef and plot_ue all work like they should.

File: viziphant/rasterplot.py (hunk 1 above)
Comment (Qkley) [reply viziphant/rasterplot.py]: Point taken. This was done as a compromise.

File: setup.py (hunk 3 above)
Comment (Qkley) [reply setup.py]: Addressed with 25d70c9ec1d09f24d1d6e54b5764c8985f99f671

Comment (Qkley) [reply setup.py]: I have also added python>=3.6 to the requirements.txt in 3eef25a99806a18760d48af5357318ed86b3e06d

File: requirements/requirements.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
+python>=3.6
```
Comment (dizcza) [inline requirements/requirements.txt]: python is not a pip package
hence, it must not be here
we should specify this somewhere in the docs (or at least in readme) as I did in https://github.com/INM-6/viziphant-new/blob/master/doc/install.rst#prerequisites

File: viziphant/unitary_event_analysis.py [hunk 5]
```diff
@@ -1,29 +1,13 @@
-
+"""
+A simple plotting function for pairwise unitary event analysis results.
```
Comment (mdenker) [inline viziphant/unitary_event_analysis.py]:
```suggestion
Standard plot function for pairwise unitary event analysis results resembling the original publication.
```

File: viziphant/unitary_event_analysis.py [hunk 6]
```diff
@@ -70,79 +44,94 @@
...
+        to the one used to generate Js_dict.
+    winsize : quantities.Quantity
+        The size of the window of analysis. This value should be identical to the one
+        used to generate Js_dict.
+    winstep : quantities.Quantity
+        The size of the window step. This value should be identical to the one used to
+        generate Js_dict.
+    pattern_hash : list of int
+        List of interested patterns in hash values. This value should be identical to
+        the one used to generate Js_dict
```
Comment (mdenker) [inline viziphant/unitary_event_analysis.py]:
```suggestion
        the one used to generate Js_dict.
```

File: requirements/requirements.txt (hunk 4 above)
Comment (Qkley) [reply requirements/requirements.txt]: I reverted back to the previous commit.

File: README.md [hunk 7]
```diff
@@ -14,7 +14,13 @@ git clone https://github.com/INM-6/viziphant.git
 cd viziphant
```
[the remark this answers is not in the document]
Comment (Qkley) [reply README.md]: Fixed. I successfully installed viziphant using ..@release-0.1 with the latest commits.

File: setup.py [hunk 8]
```diff
@@ -1,6 +1,44 @@
...
-setup(name='viziphant',
-      version='0.0.1',
-      packages=['viziphant']
-      )
+import os
+
+from setuptools import setup
+
+with open(os.path.join(os.path.dirname(__file__),
+                       "viziphant", "VERSION")) as version_file:
```
[the remark this answers is not in the document]
Comment (Qkley) [reply setup.py]: Fixed. Latest commits contain the VERSION file which displays the version upon installation.

**Pull request merged**